apps/coscrad-frontend/src/components/resources/media-items/media-item-detail.full-view.presenter.tsx

```diff
@@ -22,11 +24,15 @@ export const MediaItemDetailFullViewPresenter = ({
                     {formatBilingualText(title, titleEnglish)}
                 </div>
                 <Divider id="detail-divider" />
-
-                <video className="video-player" controls>
-                    <source src={url} type="video/webm" />
-                    <source src={url} type="video/mp4" />
-                </video>
+                {[MIMEType.png, MIMEType.jpg, MIMEType.bmp, MIMEType.svg].includes(mimeType) ? (
+                    // TODO How do we handle making this responsive?
```

Comment:
Get Justin's or Geoff's help on this.

Reply:
We might want to future-scope this though, since we don't expose the media dashboard to users yet.